keps/sig-cluster-lifecycle/kubeadm/20190916-kubeadm-operator.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Kubeadm operator
+authors:
+  - "@fabriziopandini"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+  - sig-cluster-lifecycle=
+reviewers:
+  - "@neolit123"
+  - "@rosti"
+  - "@ereslibre"
+  - "@detiber"
+  - "@vincepri"
+  - "@yastij"
+  - "@chuckha"
+approvers:
+  - "@timothysc"
+  - "@luxas"
```

Comment:
is luxas still around for this? I can't remember his timeline

Reply:
He show up sometimes at the sig meeting, and I would prefer to keep it into the loop also for kubeadm related KEPs.